### PR TITLE
feat: add alwaysTriggerPointermoveEvent option in Canvas

### PR DESCRIPTION
**Pull request by xiaoiver**
### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / Document optimization
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Performance improvement
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution

希望 pointermove 事件在 Canvas 之外也能持续触发，否则 pan 这样的手势无法做到与 hammer.js 表现保持一致。

因此增加了一个配置项 `alwaysTriggerPointerEventOnCanvas`，在创建画布时可以传入：

```js
const canvas = new Canvas({
  container: 'container',
  width: 600,
  height: 500,
  renderer: canvasRenderer,
  alwaysTriggerPointerEventOnCanvas: true,
});
```

开启后 `pointermove / up / cancel` 也会在除 `<canvas>` 之外的元素上触发：

```js
canvas.addEventListener('pointermove', () => {});
```

参考 hammer.js 的做法：https://github.com/hammerjs/hammer.js/blob/master/src/input/pointerevent.js#L30-L31

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: packages/g-lite/src/services/EventService.ts [hunk 1]
```diff
@@ -767,6 +789,7 @@ export class EventService {
     from: FederatedPointerEvent,
     type?: string,
     target?: IEventTarget,
+    fallbackTarget?: IEventTarget,
```
Comment (zengyue) [inline packages/g-lite/src/services/EventService.ts:792]: 感觉 fallbackTarget 用 currentTarget 会更合适吧

File: packages/g-lite/package.json [hunk 2]
```diff
@@ -47,6 +47,9 @@
     "@types/offscreencanvas": "^2019.6.1",
     "@types/rbush": "^3.0.0"
   },
+  "peerDependencies": {
```
Comment (xiaoiver) [inline packages/g-lite/package.json:50]: 这里应该去掉

**Pull request merged**